.github/workflows/build-app-release.yml

```
@@ -0,0 +1,150 @@
+    # These numbers will likely change when we move to EAS Update
+    - cron: "0 18 * * 2" # Tuesdays at 11 AM PT
+    - cron: "0 21 * * 4" # Thursdays at 2 PM PT
+  workflow_dispatch:
+
+jobs:
+  build-android:
+    runs-on: ubuntu-latest
+    environment: Production
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)